### DISCUSSION TITLE
Automatically Use as-is for tracks with low match %?

### DISCUSSION THREAD

**Discussion by agargara** [beetbox/beets#4905] [Q&A] [answered]
I have a directory of music which is already fairly well-tagged (carefully tagged by hand by myself.) Many of the albums are obscure and not in the MusicBrainz database. However, I still want to auto-tag the ones that are.

Is there a setting to automatically select "Use as-is" for any albums with low match % candidates? For example, I want to:
Match % 0-60: Use As-Is 
Match % 60-90: Manually check
Match % 90-100: AutoTag

**Comment by agargara** [comment]
I'm still wondering about the above, but I'll note that as a workaround for now I've decided to split it up into four steps:

1. Import with -ql flag to automatically skip albums with low match %
2. Manually move albums that were skipped but I know already have good tags into another folder
3. Import that folder with -A flag
4. Try to autotag the remainders

**Comment by Serene-Arc** [comment, marked as answer]
The part of the docs that you want is [this part](https://beets.readthedocs.io/en/latest/reference/config.html#none-rec-action). The `none_rec_option` is used when there are no recommendations for a match. You can set it in your configuration to `asis`.

You can refer to [this part](https://beets.readthedocs.io/en/latest/reference/config.html#autotagger-matching-options) of the docs to set the points at which each standard is met. You want something like this:

```yaml
import:
	none_rec_action: asis

match:
	strong_rec_thresh: 0.1
	medium_rec_thresh: 0.4
```

Strong recommendations are automatically accepted, so this will use the autotagger for anything at 90% match or above.

**Reply by agargara** [reply]
Thanks, this is exactly what I was looking for!
